src/docs/sdk/features.mdx

```diff
@@ -231,6 +231,57 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+### HTTP Client Errors
+
+The SDK automatically captures HTTP Client errors and sends them to [sentry.io](https://sentry.io).
+
+The HTTP Client integration should have 3 configuration options:
+
+- `captureFailedRequests` defaults to `false` due to PII reasons.
+  - The SDK will only capture HTTP Client errors if it is enabled.
```

Comment:
`l`: Having a list with only one item kinda looks weird. I think we could just merge them into one bullet point. This applies to all items in this list.
```suggestion
- `captureFailedRequests` defaults to `false` due to PII reasons. The SDK will only capture HTTP Client errors if it is enabled.
```

Reply:
Since this is `l` and it's the develop docs and not user-facing docs, I'd keep it as it is.